charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -55,7 +55,9 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
+        {{ if .Values.controllerManager.apiserverSkipVerify -}}
         - "--service-catalog-insecure-skip-verify=true"
+        {{- end }}
```

Comment:
Overall question:
Do we need an issue to make sure every single flag is helm exposed?